grpc/src/main/java/com/linecorp/armeria/server/grpc/GrpcServiceBuilder.java

```diff
@@ -65,7 +65,7 @@
 public final class GrpcServiceBuilder {
 
     private static final Set<SerializationFormat> DEFAULT_SUPPORTED_SERIALIZATION_FORMATS =
-            ImmutableSet.of(GrpcSerializationFormats.PROTO, GrpcSerializationFormats.PROTO_WEB);
+            GrpcSerializationFormats.values();
```

Comment:
Changed to support all `GrpcSerializationFormats` by default because there's no regression on performance and @ikhoon sometimes forgets about specifying JSON format when he's doing some test. 🤣  /cc @anuraaga